### DISCUSSION TITLE
[Closed] Virtual Machine and Other BIOS

### DISCUSSION THREAD

**Discussion by GttzRapero219** [86Box/86Box#4893] [General]
This is 86Box can emulated BIOS List
- [x] Virtual PC (4.x / 5.x / 2004 / 2007, 2007 already emulated)
- [ ] VMware
- [ ] VirtualBox
- [ ] SeaBIOS
- [x] GlaBIOS
- [ ] Bochs

**Comment by dhrdlicka** [comment]
We currently have no plans to emulate any of the listed hypervisor BIOS implementations.

**Reply by GttzRapero219** [reply]
Yes.
